cves/kernel/CVE-2022-1671.yml

```diff
@@ -75,7 +79,8 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: 
+  - https://github.com/torvalds/linux/commit/ff8376ade4f668130385839cef586a0990f8ef87
```

Comment:
If this is just the same as the fix commit, maybe its best to leave this blank if there is no official bug report.

Reply:
The github link includes an initial bug report in its comments including a stacktrace.